documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -8,6 +8,11 @@
 //
 :ispn_version: 11.0.1.Final
 
+//
+// Spring Boot attributes
+//
+:sb_version: ${version.infinispan.starter}
```

Comment:
@pruivo can we set the sb version automagically like we do for ispn_version? it needs to go into the jenkins release file but I'm not sure how to add it or if it's possible.

Reply:
you don't need the new attribute. the version is `ispn_version`

Reply:
YES!!